### PR TITLE
Add a note about httpclient

### PR DESCRIPTION
**Pull request by imba-tjd**
You can change the sentence where necessary, because my native language is not English.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@imba-tjd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/containers/tutorial-multicontainer.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ Add a project to the same solution and call it *MyWebAPI*. Select **API** as the
        }
     }
    ```
+    > [!NOTE]
+    > You shouldn't dispose `HttpClient` everytime finishing request. This code is for tutorial only .For the best way to use HttpClient, see [Use HttpClientFactory to implement resilient HTTP requests](https://docs.microsoft.com/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests)
```
Comment (ghogen) [inline docs/containers/tutorial-multicontainer.md]: How about:
In real-world code, you shouldn't dispose `HttpClient` after every request. For best practices, see...

**Comment by PRMerger9** [comment]
@imba-tjd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/containers/tutorial-multicontainer.md (hunk 1 above)
Comment (imba-tjd) [reply docs/containers/tutorial-multicontainer.md]: done.

**Comment by ghogen** [comment]
#sign-off

**Comment by PRMerger10** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**